CHANGES.md

```diff
@@ -40,8 +40,10 @@ To be released.
  -  `BlockChain<T>.MineBlock()` and `BlockChain<T>.GetNextTxNonce()` methods
     became to ignore transactions that didn't follow `Transaction<T>.Nonce`
     sequentially and treat them as pendings.  [[#365]]
- - `BlockChain<T>` became to evaluate `IBlockPolicy<T>.BlockAction` and set the
-   state when a block is appended to the chain.  [[#319], [#367]]
+ -  `BlockChain<T>` became to evaluate `IBlockPolicy<T>.BlockAction` and set the
+    state when a block is appended to the chain.  [[#319], [#367]]
+ -  `PreloadAsync()` became to skip state synchronization from
+    `trustedStateValidators` if `render` is true.
```

Comment:
This change should not be logged because `trustedStateValidators` option will be introduced 0.5.